sphinx.po

```diff
@@ -251,9 +246,8 @@ msgid "Python 3.5 (security-fixes)"
 msgstr "Python 3.5 (correctifs de sécurité)"
 
 #: tools/templates/indexsidebar.html:11
-#, fuzzy
 msgid "Python 2.7 (EOL)"
-msgstr "Python 2.7 (stable)"
+msgstr "Python 2.7 (obsolète)"
```

Comment:
```suggestion
msgstr "Python 2.7 (en fin de vie)"
```
?

Reply:
Wikipédia dit « fin de vie »

- https://en.wikipedia.org/wiki/End-of-life_(product)
- https://fr.wikipedia.org/wiki/Fin_de_vie_(produit)